list.js

```diff
@@ -1,6 +1,57 @@
 (function() {
     var events = [];
     var ITEMS_ON_PAGE = 6;
+    var querySet = [];
+
+    var index = lunr(function() {
+        this.ref('id');
+        this.field('title', {boost: 10});
+        this.field('description', {boost: 5});
+    });
+
+    var doSearch = function() {
+        var q = $('#q').val();
+        var results = index.search(q);
+        var $el = $('#calendarList');
+        $el.empty();
+        $el.show();
+        $el.append('<div class="arrow"></div>');
+        $el.append(
+            $('<h2>RESULTS FOR: "' + q + '"</h2>')
+        );
+        if (results.length === 0) {
+            $el.append('<div class="q-no-item">Unfortunately, there are ' +
+                'no results matching what you\'re looking for in ' +
+                'the Columbia Film Glossary content.</div>');
+        } else {
+            var searchResults = [];
+            for (var r in results) {
+                var e = events[results[r].ref];
+                searchResults.push(e);
+            }
+            querySet = searchResults;
```

Comment:
Similarly, instead of altering the global `querySet`, maybe the search function could just return the array of search results, then the caller sets this. But then, I see you're doing DOM manipulation here, as well -- so I think I'm going to break this function up into two different ones.